sudukoValidation/index.js

```diff
@@ -0,0 +1,84 @@
+class Suduko {
+    constructor() {
+        this.matrix;
+        this._col
+        this._rows
+        this._grid
+    }
+
+    init(arr) {
```

Comment:
why do we need a separate function called init? We should have utilized constructor instead.

init is constructor in languages like java and swift but this function creates confusion in js.